code/src/index.css

```diff
@@ -1,13 +1,84 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
+  font-family: "Roboto Mono", monospace;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.message-card {
+margin: 25px;
+border: 1px solid;
+padding: 25px;
+box-shadow: 5px 5px 0px 0px #000;
+width: 300px;
```

Comment:
Maybe you did not want any media-queries and Im not sure if we were supposed to since not mentioned in blue requirements. Even though it looks nice in all devices, the card could have been a little bit wider in desktop mode. In order not to add fixed pixels for the card you've could have used the 
`margin : 0 auto ` for the body.
 And then a width in percentage for the message.card  + maybe a max-width: 
`max-width: 500px;
  width: 80%;`